frontend/src/components/ScheduleCreation.vue

```diff
@@ -314,7 +323,12 @@ const saveSchedule = async (withConfirmation = true) => {
   emit('created');
   // Update our reference schedule!
   referenceSchedule.value = { ...scheduleInput.value };
-  resetSchedule(false);
+  revertForm(false);
+};
+
+// Update slug with a random 8 character string
+const refreshSlug = () => {
+  slugInput.value = (Math.random() + 1).toString(36).substring(4);
```

Comment:
We can probably substring https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID

Reply:
That's better, thanks!